### ISSUE TITLE
should normalize over all elements?

### ISSUE THREAD

**Issue by lucast4** [rfeinman/pyBPL#5]
https://github.com/rfeinman/pyBPL/blob/482d3dd5c809a5a854ee78dc19963bab6a10ac38/pybpl/library/spatial_OLD/spatial_hist.py#L74

I think this line should be soemthign like this?
        logpN = logN - torch.logsumexp(logN.reshape(-1), 0)

**Comment by rfeinman** [comment]
Thank you, just fixed this. Curious though how the assertion of the next line still passes...

**Comment by lucast4** [comment]
yeah actually I only noticed this because it failed for me...
